cwa/olin/src/lib.rs

```diff
@@ -1,54 +1,92 @@
+/// Main classes to perform the low-level operations
+/// described in the Olin specs. In particular the basic
+/// "actions": open, read, write, close, flush on files.
+/// 
+/// Modules:
+/// * sys
+/// * err
+/// * log
+/// * env
+/// * runtime
+/// * startup
+/// * time
+/// * stdio
+/// * random
+///
+/// ```Resource``` class defines a generic class for a resource
+/// with methods bound for FFI access
+
 extern crate chrono;
 
 use std::io::{self, Read, Write};
 
+/// import HTTP client from local
+/// TODO: couldn't get how this is used as it seems not to be called anywhere
```

Comment:
https://github.com/Xe/olin/blob/master/cwa/tests/src/olintest/http.rs

Reply:
so it is not used in `lib.rs`?